internal/nodes/avi_model_nodes.go

```diff
@@ -892,6 +892,7 @@ type AviVSVIPNode struct {
 	SubnetIP                string
 	SubnetPrefix            int32
 	NetworkNames            []string
+	BGPPeerLabels           *[]string
```

Comment:
Why is this a pointer to a slice and not just a slice of string? Are you planning to pass this variable around and manipulate them inside functions? Slices should otherwise always be pass by reference.